2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_setup.py:_flush():71] setting env: {}
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_init.py:_log_setup():371] Logging user logs to D:\rushan\II\lab3\II_Group1_Lab3\wandb\run-20211211_170848-12ccal4g\logs\debug.log
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_init.py:_log_setup():372] Logging internal logs to D:\rushan\II\lab3\II_Group1_Lab3\wandb\run-20211211_170848-12ccal4g\logs\debug-internal.log
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000021234B57D48>
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_init.py:init():404] calling init triggers
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 3qh2x7u7
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_run.py:finish():1463] finishing run marvink/lab3/3qh2x7u7
2021-12-11 17:08:48,620 INFO    MainThread:7704 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2021-12-11 17:08:48,635 INFO    MainThread:7704 [wandb_run.py:_restore():1758] restore
2021-12-11 17:08:49,241 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 791
}

2021-12-11 17:08:49,507 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 791
}

2021-12-11 17:08:50,522 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 36514
}

2021-12-11 17:08:50,632 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 36767
}

2021-12-11 17:08:50,741 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36767
  total_bytes: 36767
}

2021-12-11 17:08:50,850 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36767
  total_bytes: 36767
}

2021-12-11 17:08:50,960 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36767
  total_bytes: 36767
}

2021-12-11 17:08:51,069 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36767
  total_bytes: 36767
}

2021-12-11 17:08:51,178 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36767
  total_bytes: 36767
}

2021-12-11 17:08:51,569 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36767
  total_bytes: 36767
}

2021-12-11 17:08:51,959 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36767
  total_bytes: 36767
}
local_info {
}

2021-12-11 17:08:53,100 INFO    MainThread:7704 [wandb_run.py:_append_history():2134] rendering history
2021-12-11 17:08:53,100 INFO    MainThread:7704 [wandb_run.py:_append_summary():2089] rendering summary
2021-12-11 17:08:53,100 INFO    MainThread:7704 [wandb_run.py:_append_files():2184] logging synced files
2021-12-11 17:08:53,115 INFO    MainThread:7704 [wandb_init.py:init():449] starting backend
2021-12-11 17:08:53,115 INFO    MainThread:7704 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2021-12-11 17:08:53,115 INFO    MainThread:7704 [backend.py:ensure_launched():199] starting backend process...
2021-12-11 17:08:53,178 INFO    MainThread:7704 [backend.py:ensure_launched():205] started backend process with pid: 2740
2021-12-11 17:08:53,178 INFO    MainThread:7704 [wandb_init.py:init():458] backend started and connected
2021-12-11 17:08:53,178 INFO    MainThread:7704 [wandb_run.py:_label_probe_notebook():917] probe notebook
2021-12-11 17:08:53,178 INFO    MainThread:7704 [wandb_run.py:_label_probe_notebook():927] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-12-11 17:08:53,178 INFO    MainThread:7704 [wandb_init.py:init():519] updated telemetry
2021-12-11 17:08:53,992 INFO    MainThread:7704 [wandb_init.py:init():549] communicating current version
2021-12-11 17:08:55,164 INFO    MainThread:7704 [wandb_init.py:init():554] got version response 
2021-12-11 17:08:55,164 INFO    MainThread:7704 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-11 17:08:55,414 INFO    MainThread:7704 [wandb_init.py:init():592] starting run threads in backend
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_run.py:_console_start():1816] atexit reg
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_init.py:init():619] run started, returning control to user process
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_config.py:__setitem__():144] config set batch_size = 256 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000021238380E88>>
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_run.py:_config_callback():962] config_cb batch_size 256 None
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_config.py:__setitem__():144] config set epochs = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000021238380E88>>
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_run.py:_config_callback():962] config_cb epochs 10 None
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_config.py:__setitem__():144] config set lr = 0.005 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000021238380E88>>
2021-12-11 17:08:58,866 INFO    MainThread:7704 [wandb_run.py:_config_callback():962] config_cb lr 0.005 None
2021-12-12 05:27:53,250 INFO    MainThread:7704 [wandb_init.py:_pause_backend():284] pausing backend
2021-12-12 05:27:53,250 INFO    MainThread:7704 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-12-12 09:10:03,970 INFO    MainThread:7704 [wandb_init.py:_resume_backend():293] resuming backend
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_setup.py:_flush():71] setting env: {}
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_init.py:_log_setup():371] Logging user logs to D:\rushan\II\lab3\II_Group1_Lab3\wandb\run-20211212_091004-ktwmavcr\logs\debug.log
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_init.py:_log_setup():372] Logging internal logs to D:\rushan\II\lab3\II_Group1_Lab3\wandb\run-20211212_091004-ktwmavcr\logs\debug-internal.log
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_init.py:init():404] calling init triggers
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 12ccal4g
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_run.py:finish():1463] finishing run marvink/lab3/12ccal4g
2021-12-12 09:10:04,720 INFO    MainThread:7704 [jupyter.py:save_history():429] not saving jupyter history
2021-12-12 09:10:04,720 INFO    MainThread:7704 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2021-12-12 09:10:04,720 INFO    MainThread:7704 [wandb_run.py:_restore():1758] restore
2021-12-12 09:10:04,954 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 791
}

2021-12-12 09:10:05,188 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 791
}

2021-12-12 09:10:05,454 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 36471
}

2021-12-12 09:10:05,563 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 791
  total_bytes: 36725
}

2021-12-12 09:10:05,673 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:05,782 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:05,892 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:05,992 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,102 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,211 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,320 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,430 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,539 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,648 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,758 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11936
  total_bytes: 36725
}

2021-12-12 09:10:06,867 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36725
  total_bytes: 36725
}

2021-12-12 09:10:06,976 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36725
  total_bytes: 36725
}

2021-12-12 09:10:07,086 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36725
  total_bytes: 36725
}

2021-12-12 09:10:07,195 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36725
  total_bytes: 36725
}

2021-12-12 09:10:07,304 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36725
  total_bytes: 36725
}

2021-12-12 09:10:08,585 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36725
  total_bytes: 36725
}

2021-12-12 09:10:08,992 INFO    MainThread:7704 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 36725
  total_bytes: 36725
}
local_info {
}

2021-12-12 09:10:10,116 INFO    MainThread:7704 [wandb_run.py:_append_history():2134] rendering history
2021-12-12 09:10:10,116 INFO    MainThread:7704 [wandb_run.py:_append_summary():2089] rendering summary
2021-12-12 09:10:10,116 INFO    MainThread:7704 [wandb_run.py:_append_files():2184] logging synced files
